src/promptflow-tools/CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # Release History
 
-## 1.4.0 (Upcoming)
+## 1.4.0 (2024.03.22)
 
 ### Features Added
 - Enable token based auth in azure openai connection for tools.
 - Add "seed" to LLM and GPT-Vision tool inputs.
 
+### Improvements
+- Improve error message when LLM tool meets gpt-4-vision-preview model
+
+### Bugs Fixed
+- Fix dynamic list function for GPT-Vision tool to avoid misleading errors
```

Comment:
Set default values to workspace triad parameters of dynamic list function to avoid misleading errors when clicking tool input.